sql/parser/ast.go

```diff
@@ -205,22 +308,33 @@ const (
 	astDropTable       = "DROP TABLE"
 	astDropView        = "DROP VIEW"
 	astRenameTable     = "RENAME TABLE"
+	astShowDatabases   = "SHOW DATABASES"
 	astShowTables      = "SHOW TABLES"
 	astShowIndex       = "SHOW INDEX FROM"
 	astShowColumns     = "SHOW COLUMNS FROM"
 	astShowFullColumns = "SHOW FULL COLUMNS FROM"
 	astTruncateTable   = "TRUNCATE TABLE"
+	astIfNotExists     = " IF NOT EXISTS"
+	astUnique          = "UNIQUE"
+	astPrimaryKey      = "PRIMARY KEY"
+	astKey             = "KEY"
+	astUnsigned        = "UNSIGNED"
 )
 
 func (node *DDL) String() string {
 	switch node.Action {
-	case astCreateIndex, astDropIndex:
+	case astDropIndex:
 		return fmt.Sprintf("%s %s ON %s", node.Action, node.Name, node.NewName)
 	case astRenameTable:
 		return fmt.Sprintf("%s %s %s", node.Action, node.Name, node.NewName)
-	case astCreateDatabase, astCreateTable, astCreateView:
-		return fmt.Sprintf("%s %s", node.Action, node.NewName)
+	case astCreateView:
+		return fmt.Sprintf("%s%s %s", node.Action, node.IfNotExists, node.NewName)
+	case astShowDatabases:
```

Comment:
@jusongchen Cockroach will have the concept of a group of tables. Whether that is called a database, schema, catalog, namespace, directory or something else is still up in the air. The term database certainly has different meanings between DBs. Catalog feels like an obscure term to me. Directory is compelling due to the similarity to a filesystem directory.